src/zx_graph.jl

```diff
@@ -184,26 +227,31 @@ end
 
 function show(io::IO, zxg::ZXGraph{T}) where {T<:Integer}
     println(io, "ZX-graph with $(nv(zxg)) vertices and $(ne(zxg)) edges:")
-    for e in edges(zxg.mg)
-        print(io, "(")
-        print_spider(io, zxg, src(e))
-        if is_hadamard(e)
-            printstyled(io, " <-> "; color = :blue)
-        else
-            print(io, " <-> ")
+    vs = sort!(spiders(zxg))
+    for i = 1:length(vs)
+        for j = i+1:length(vs)
+            if has_edge(zxg, vs[i], vs[j])
+                print(io, "(")
+                print_spider(io, zxg, vs[i])
+                if is_hadamard(zxg, vs[i], vs[j])
+                    printstyled(io, " <-> "; color = :blue)
+                else
+                    print(io, " <-> ")
+                end
+                print_spider(io, zxg, vs[j])
+                print(io, ")\n")
+            end
         end
-        print_spider(io, zxg, dst(e))
-        print(io, ")\n")
     end
 end
 
 function rounding_phases!(zxg::ZXGraph{T, P}) where {T<:Integer, P}
```

Comment:
I should have comment here, this should be a verb `round_phases`, I think the current name is not very correct in grammar